src/main/java/junit/framework/TestFailure.java

```diff
@@ -10,8 +10,8 @@
  * @see TestResult
  */
 public class TestFailure {
-    protected Test fFailedTest;
-    protected Throwable fThrownException;
+    private Test fFailedTest;
```

Comment:
This change could break users' code (for example, subclasses that reference this field)